### PR TITLE
ROC-3481: fix bug

### PR DESCRIPTION
**Pull request by yarpo**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/ROC-3481

### Change description ###
On sealed claim and claimant copy pdf total amount should always be the same.

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: domain-model/src/main/java/uk/gov/hmcts/cmc/domain/models/InterestDate.java [hunk 1]
```diff
@@ -60,6 +60,21 @@ public InterestEndDateType getEndDateType() {
         return endDateType;
     }
 
+    @JsonIgnore
```
Comment (timja-kainos) [inline domain-model/src/main/java/uk/gov/hmcts/cmc/domain/models/InterestDate.java]: hmm?

File: src/main/java/uk/gov/hmcts/cmc/claimstore/documents/ClaimDataContentProvider.java [hunk 2]
```diff
@@ -97,12 +99,15 @@ public ClaimContent createContent(Claim claim) {
...
-            amountBreakDown.getRows()
-                .stream()
-            .filter(row -> row != null && row.getAmount() != null)
-            .map(AmountRowContent::new)
-            .collect(Collectors.toList())
+            removeEmptyRows(amountBreakDown.getRows())
         );
     }
 
+    private static List<AmountRowContent> removeEmptyRows(List<AmountRow> rows) {
```
Comment (timja-kainos) [inline src/main/java/uk/gov/hmcts/cmc/claimstore/documents/ClaimDataContentProvider.java]: it does more than remove empty rows, it converts to a format for display

File: domain-model/src/main/java/uk/gov/hmcts/cmc/domain/models/InterestDate.java (hunk 1 above)
Comment (yarpo) [reply domain-model/src/main/java/uk/gov/hmcts/cmc/domain/models/InterestDate.java]: We don't want it to be mapped into JSON.

File: src/main/java/uk/gov/hmcts/cmc/claimstore/documents/ClaimDataContentProvider.java [hunk 3]
```diff
@@ -97,12 +99,15 @@ public ClaimContent createContent(Claim claim) {
...
-            .map(AmountRowContent::new)
-            .collect(Collectors.toList())
+            mapToAmountRowContent(amountBreakDown.getRows())
         );
     }
 
+    private static List<AmountRowContent> mapToAmountRowContent(List<AmountRow> rows) {
+        return rows
+            .stream()
+            .filter(row -> row != null && row.getAmount() != null)
```
Comment (sabahirfan) [inline src/main/java/uk/gov/hmcts/cmc/claimstore/documents/ClaimDataContentProvider.java]: You may use  streams filter here. 'filter(Objects::nonNull)'

**Pull request merged**